Misc/NEWS.d/next/Build/2023-01-21-10-31-35.gh-issue-101152.xvM8pL.rst

```diff
@@ -0,0 +1 @@
+Added the PEP-699 Implementation
```

Comment:
Try to be more descriptive. You should say what this means and what are the effects on external code. For example, how would 3rd party code be affected?

Also please add this to the whats new in 3.12 document.

Reply:
yes I'll add it